src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/AdvSimd.cs

```diff
@@ -9034,6 +9184,111 @@ public new abstract class Arm64 : ArmBase.Arm64
         /// </summary>
         public static unsafe Vector128<ulong> LoadVector128(ulong* address) => LoadVector128(address);
 
+        /// <summary>
+        ///   A64: LD1 { Vn.16B, Vn+1.16B }, [Xn]
```

Comment:
@tannergooding I think the noted register in this comment should be:
```csharp
///   A64: LD1 { Vn.8B, Vn+1.8B }, [Xn]
```
as well as adjusting the other noted registers in the comments on the other `LoadVector64x2/3/4*` methods for `sbyte`,`short`,`ushort`,`long`,`ulong`,`float`.

```csharp
///   A64: LD1 { Vn.4H, Vn+1.4H }, [Xn]
```
```csharp
///   A64: LD1 { Vn.2S, Vn+1.2S }, [Xn]
```

https://developer.arm.com/documentation/102159/0400/Load-and-store---data-structures

Reply:
yes, this should be half the sizes i.e. 8B, 4H, etc.

Reply:
Looks like it, yeah. I definitely might've made a mistake in documenting some of the things. Lots of APIs and lots of copy/pasting